src/chain/sanitizers.ts

```diff
@@ -10,6 +10,7 @@ export interface Sanitizers<Return> {
   ltrim(chars?: string): Return;
   normalizeEmail(options?: Options.NormalizeEmailOptions): Return;
   rtrim(chars?: string): Return;
+  replace(new_value: any, values_to_replace?: any[]): Return;
```

Comment:
I wonder if having `values_to_replace` be optional won't make it confusing, e.g.

```js
check('name').replace('John')
```

could be interpreted as "replace `name` value with `"John"`".

WDYT of swapping them?

Reply:
You mean to swap `new_value` and `values_to_replace`, right?
```typescript
replace(values_to_replace: any[], new_value: any): Return;
```
And what about if providing an empty array as `values_to_replace` would use default `[null, undefined, NaN, '']`.
```typescript
// declaration
function replace(values_to_replace: any[], new_value: any) {
  const v = values_to_replace.length !== 0 ? values_to_replace : [null, undefined, NaN, '']
  ...
}

// use case 
// would replace [null, undefined, NaN, ''] with undefined
check('name').replace([], undefined)
```

Reply:
> Another solution could be to remove the default array, and to add it as a different sanitizer called `default()` instead.

This sounds good to me! Are you willing to tackle it in this PR, or a separate one? Either way is fine.

Reply:
I think separating it may be a better choice.